zfmt/dag.go

```diff
@@ -429,32 +429,35 @@ func (c *canonDAG) op(p dag.Op) {
 			c.assignments(p.Args)
 		}
 		c.close()
-	case *dag.From:
-		// XXX cleanup for single trunk
+	case *dag.Lister:
 		c.next()
-		c.open("from (")
-		for _, trunk := range p.Trunks {
-			c.ret()
-			if trunk.Pushdown != nil {
-				c.head = true
-				c.open("(pushdown")
-				c.op(trunk.Pushdown)
-				c.write(")")
-				c.close()
-				c.ret()
-			}
-			c.write("%s", source(trunk.Source))
-			if trunk.Seq != nil && len(trunk.Seq.Ops) != 0 {
-				c.open()
-				c.head = true
-				c.write(" =>")
-				c.op(trunk.Seq)
-				c.close()
-			}
+		c.open("lister")
+		c.write(fmt.Sprintf(" pool %s commit %s", p.Pool, p.Commit))
```

Comment:
c.write calls fmt.Sprintf for you.
```suggestion
		c.write(" pool %s commit %s", p.Pool, p.Commit)
```

Reply:
I fixed all the others that were in there.